apis/bulkupload/bulkserviceapi.yaml

```diff
@@ -412,6 +412,71 @@ paths:
             error.
           schema:
             $ref: '#/definitions/CommonErrorResponse'
+  /v2/bulk/user/upload:
+    post:
+      description: >-
+        This API allows you to upload users in bulk on the Sunbird Platform. Using this API, State administrators will have the facility to upload a valid list of State users on the Sunbird Platform.
+        The functionality for this is as follows:
+        The State administrator uploads a list of valid users into a Shadow DB which is a placeholder DB. Each custodian org user identifier (email, phone) is matched with the Shadow DB. Only if there is a match is found, then that user is moved from the custodian org to the State org.
+           - The **_/v2/bulk/user/upload_** endpoint executes " bulk upload user" request based on parameters provided as metadata in the request body.
+           - It is mandatory to provide values for parameters marked with *.
+           - Mandatory fields cannot be null or empty.
+      summary: Upload Users in Bulk
+      tags:
+        - Bulk Operation APIs
+      operationId: bulk upload user req
+      produces:
+        - application/json
+      consumes:
+        - multipart/form-data
+      parameters:
+        - name: user
+          in: formData
+          required: true
+          type: file
+          format: file
+          description: >-
+            This is a .csv file.
+            - Each line of the file is a data record. Each record consists of one or more user fields, separated by commas. Each successfully created user is associated with root org of the uploader (org admin).
+
+            The CSV file format supported is as follows:
+              - Name: Name of user as per state records (mandatory)
+              - Email: Email. Either phone or email is mandatory.
+              - Phone: Phone number. Either phone or email is mandatory.
+              - External Org ID: School ID as provided in Diksha .

```

Comment:
@aishwa8141 @manzarul as discussed, please check these descriptions as we cannot use Diksha etc in Sunbird. If we make it generic, then we can add it to Sunbird documentation. Else, we will need to check where to keep this file.